worlds/alttp/Bosses.py

```diff
@@ -166,7 +166,7 @@ def GanonDefeatRule(state, player: int):
 
 
 def get_plando_bosses(boss_shuffle: str) -> List[str]:
-    options = boss_shuffle.lower().split(";")
+    options = boss_shuffle.split(";")
```

Comment:
don't you want lower() for bosses at some point? I don't see it being added in class Bosses, so removing it here would never make it lower? - i could be wrong though

Reply:
i'm doing this in from_text so doing it a second time was unneccessary